src/main.js

```diff
@@ -189,6 +189,11 @@ app.on('ready', function() {
         title: arg.title,
         content: arg.options.body
       });
+
+      /* Todo: add idle here */
```

Comment:
Not sure if this is a good place for this, since ```ipc.on('notified',``` is only a "hack" to enable baloon tooltips on windows < 8.1 if I understood it correctly.
This means, we should try to get rid of it in the log run. And I'm wondering if this does even get called for non win32?
If we refactor notifications, which isn't hopefully isn't unlikely, we might want to drop this.